internal/pkg/scaffold/stub.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scaffold
+
+import (
+	"path/filepath"
+
+	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold/input"
+)
+
+const StubFile = "stub.go"
+
+type Stub struct {
+	input.Input
+
+	Resource *Resource
+}
+
+func (s *Stub) GetInput() (input.Input, error) {
+	if s.Path == "" {
+		s.Path = filepath.Join(ApisDir, s.Resource.GoImportGroup, StubFile)
+	}
+	s.TemplateBody = stubTmpl
+	return s.Input, nil
+}
+
+const stubTmpl = `package {{.Resource.GoImportGroup}}
```

Comment:
This scaffold is related to suppressing a warning from deepcopy-gen not finding go files.

Since it's unrelated to adding modules support we can address this in another issue/PR.